nbqa/replace_source.py

```diff
@@ -88,8 +88,7 @@ def main(python_file: "Path", notebook: "Path", notebook_info: NotebookInfo) ->
     with open(notebook) as handle:
         notebook_json = json.load(handle)
```

Comment:
You could also write this using `json.loads(notebook.read_text())`

Reply:
nice!

Reply:
seems there's a _slight_ performance hit from this, but it's probably fine for the sake of readability
```python
In [8]: %timeit json.loads(Path(notebook).read_text())                                                                                                                                                    
41 µs ± 159 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)

In [9]: %timeit with open(notebook) as handle: json.load(handle)                                                                                                                                          
31.7 µs ± 129 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```